genie_registry/clinical.py

```diff
@@ -95,10 +95,10 @@ def _check_int_dead_consistency(clinicaldf: DataFrame) -> str:
     # Check that all string values are equal each other
     is_equal = all(clinicaldf.loc[is_str, "DEAD"] == clinicaldf.loc[is_str, "INT_DOD"])
     # If dead, int column can't be Not Applicable
-    # If alive, int column can't have values
+    # If alive, int column must be Not Applicable
```

Comment:
Just as a note here, I think we only need to check the interval and not the year, because we check for consistent strings between the INT and YEAR columns.  So by checking if INT = Not Applicable, the other check ensures that the YEAR = Not Applicable as well.  

Does that sound right to you?

Reply:
yes, aligns with my thinking as well